### PR TITLE
Added command to wipe fluff table

### PR DESCRIPTION
**Pull request by kkrampa**
@czue What do you think about this?

### CODE REVIEW

File: corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                                                         help='Skip important confirmation warnings.'),)
+
+    def handle(self, *args, **options):
+        pillow_class = to_function(args[0])
+        if not options['noinput']:
+            confirm = raw_input("""
+                You have requested to wipe %s table
+
+                Type 'yes' to continue, or 'no' to cancel: """ % pillow_class.__name__
+            )
```
Comment (dimagimon) [inline corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py]: E124 closing bracket does not match visual indentation

File: corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class Command(BaseCommand):
+
+    option_list = BaseCommand.option_list + (make_option('--noinput',
+                                                         action='store_true',
+                                                         dest='noinput',
+                                                         default=False,
+                                                         help='Skip important confirmation warnings.'),)
+
+    def handle(self, *args, **options):
+        pillow_class = to_function(args[0])
```
Comment (czue) [inline corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py]: nit: could use `get_pillow_by_name` to make this easier to pass in

File: corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        if not options['noinput']:
+            confirm = raw_input("""
+                You have requested to wipe %s table
+
+                Type 'yes' to continue, or 'no' to cancel: """ % pillow_class.__name__
+            )
+
+            if confirm != 'yes':
+                print "\tReset cancelled."
```
Comment (czue) [inline corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py]: nit: should be "wipe canceled" or similar

File: corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+                return
+        engine = sqlalchemy.create_engine(settings.SQL_REPORTING_DATABASE_URL)
+        table_name = 'fluff_{0}'.format(pillow_class.indicator_class.__name__)
+        table = sqlalchemy.Table(table_name, sqlalchemy.MetaData())
```
Comment (czue) [inline corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py]: is there not a helper function to get the table from the pillow? maybe we could add one?

**Comment by czue** [comment]
@kkrampa looks ok for a one-off but had a couple quick thoughts

**Comment by kkrampa** [comment]
@czue Updated

**Comment by czue** [comment]
:+1: thanks!

**Pull request merged**